python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -24,6 +24,10 @@
 # Type alias for a node id.
 NodeIdStr = str
 
+# Higher values here are better for prefetching and locality. It's ok for this to be
+# fairly high since streaming backpressure prevents us from overloading actors.
```

Comment:
Should we just set it to a large number then? It'll then take the benefits while avoid downside automatically.

Reply:
There is the risk of mis-estimation of sizes, I'd prefer to keep on the conservative end.